ci_example_python/demos/demo_ros_pid.py

```diff
@@ -0,0 +1,56 @@
+#! /usr/bin/python
+
+##@package ci_example_python
+# 
+# @file demo_ros_pid.py
+# @author Vincent Berenz
+# @author Maximilien Naveau
+# license License BSD-3-Clause
+# @copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
+# @date 2019-05-22
+#
+# @brief This demos present the API of the PID class using ROS param.
+#
+
+##@example demo_ros_pid.py
+#
+# In order to run this demos the `ci_example_python/python/ci_example_python` path
+# needs to be added in the `PYTHONPATH`. This is done after:
+# - 1. "catkin_make" is called from the root of the catkin workspace
+# - 2. "source ./devel/setup.bash" is called from the root of the catkin workspace
+# - 3. roscore is called in an additional terminal
+# - 4. Finally run `python ./demos_ros_pid.py`
```

Comment:
As this already depends on ROS, should we rather use `rosrun ci_example_python demos_ros_pid.py` here?

Reply:
Yeah, you are right I will make the change.